### PR TITLE
Flap emotes work with fairy wings again

### PR DESCRIPTION
**Pull request by Pathid**
Something that got cut with emote datums.

:cl:
* rscadd: Brought back a cut emote feature: *flap and *aflap work with fairy wings and make them animate.

### CODE REVIEW

**Comment by Kurfursten** [comment]
I think(?) Vox were able to use flap/aflap as well

**Comment by Dacendeth** [comment]
Vox were indeed able to flap previously.

**Comment by Pathid** [comment]
Vox can. It's non-vox humanoids and non-mommi borgs that lost that ability, and yeah that was done kinda unatomically. (Aren't vox wings metaphorical anyway?)

**Comment by ShiftyRail** [comment]
I always though that aflap was just for Vox & MoMMis
I apologise for the unatomacity, thanks Panthid for taking care of this

This closes #19383

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -97,6 +97,20 @@
 		var/mob/living/carbon/human/H = user
 		if (isvox(H) || isskelevox(H))
 			return TRUE
+		if(H.wear_suit && istype(H.wear_suit,/obj/item/clothing/suit/clownpiece))
```
Comment (DamianX) [inline code/modules/mob/living/emote.dm]: `H.is_wearing_item(/obj/item/clothing/suit/clownpiece, slot_wear_suit)`

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -97,6 +97,20 @@
...
+			return TRUE
+
+/datum/emote/living/flap/run_emote(mob/user, params)
+	. = ..()
+	if(. && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.wear_suit && istype(H.wear_suit,/obj/item/clothing/suit/clownpiece))
+			var/obj/item/clothing/suit/clownpiece/wings = H.wear_suit
```
Comment (DamianX) [inline code/modules/mob/living/emote.dm]: `var/obj/item/clothing/suit/clownpiece/wings = H.is_wearing_item(/obj/item/clothing/suit/clownpiece, slot_wear_suit)`

File: code/modules/mob/living/emote.dm [hunk 3]
```diff
@@ -97,6 +97,20 @@
...
 		if (isvox(H) || isskelevox(H))
 			return TRUE
+		if(H.wear_suit && istype(H.wear_suit,/obj/item/clothing/suit/clownpiece))
+			return TRUE
+
+/datum/emote/living/flap/run_emote(mob/user, params)
+	. = ..()
+	if(. && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.wear_suit && istype(H.wear_suit,/obj/item/clothing/suit/clownpiece))
```
Comment (DamianX) [inline code/modules/mob/living/emote.dm]: change to `if(wings)` after the other change

**Comment by DamianX** [comment]
Proper way would be to restore the broken on_emote event handler but I'm not gonna ask you to do that

**Pull request merged**